devices/gpu/nvidia/device.go

```diff
@@ -149,6 +159,10 @@ func (d *NvidiaDevice) SetConfig(cfg *base.Config) error {
 // Fingerprint streams detected devices. If device changes are detected or the
 // devices health changes, messages will be emitted.
 func (d *NvidiaDevice) Fingerprint(ctx context.Context) (<-chan *device.FingerprintResponse, error) {
+	if !d.enabled {
+		return nil, errDeviceNotEnabled
```

Comment:
I'm not 100% clear on what triggers the fingerprint call... are we going to end up spamming the logs if we return an error here: https://github.com/hashicorp/nomad/blob/v0.12.0/client/devicemanager/instance.go#L340 or does this just happen once?

(And should it be logged as an error at all if we disabled it?)

Reply:
When fingerprinting, the device manager gives up and doesn't fingerprinting again.  So we only log it once.

It's logged as an error like the following:

```
    2020-07-21T13:27:43.558Z [ERROR] client.device_mgr: fingerprinting failed: plugin=nvidia-gpu error="Nvidia device is not enabled"
```

I can make it a debug/info statement.

Reply:
Made it info in 7896d1f8a9097936e03162ca0e73341a370f7d7a .

Reply:
LGTM!